src/rmarkdown/preview.ts

```diff
@@ -86,10 +86,16 @@ class RMarkdownPreview extends vscode.Disposable {
         this.htmlLightContent = $.html();
 
         // make the output chunks a little lighter to stand out
-        const chunkCol = String(config().get('rmarkdown.chunkBackgroundColor'));
-        const colReg = /[0-9.]+/g;
-        const regOut = chunkCol.match(colReg);
-        const outCol = `rgba(${regOut[0] ?? 100}, ${regOut[1] ?? 100}, ${regOut[2] ?? 100}, ${Number(regOut[3]) + 0.05 ?? .5})`;
```

Comment:
I suspect that the alpha channel should be something like `${Number(regOut[3]) - 0.05 ?? .05}` to make the output more like the background. In light themes, the default rendering looks like the code chunk is more highlighted than the output from the background.